packages/IMAPClient-Core.package/ICCommand.class/class/join..st

```diff
@@ -0,0 +1,6 @@
+commands
+join: aCommandArray
+
+	^ aCommandArray asOrderedCollection
+			addFirst: ICEndPoint protocolCommandStart;
```

Comment:
Why are those constants not moved from `ICEndpoint` to the `ICCommand` class?

Reply:
Because they are needed for parsing and file storage too. If we put them into `ICCommand` we would have to refactor them in `ICEndPoint`. We decided to not do so, because we don't want to use `ICCommand` constants while parsing

Reply:
I'd suggest to put these constants either in `ICParser` or `ICCommand`. `ICEndPoint` is the wrong place for it in my opinion